### PR TITLE
Cache gets both sha1 and murmur, but only stores in sha1.

### PR DESCRIPTION
**Pull request by Nifty255**

### CODE REVIEW

**Review by cube-drone — APPROVED** [review]
I give this the LGTM of approval

File: lib/Tance.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+var crypto = require('crypto');
```
Comment (cube-drone) [inline lib/Tance.js:3]: var? VAR??!?!

**Review by cube-drone — APPROVED** [review]
image
mamma mia, that's a spicy-a cache

File: lib/Tance.js (hunk 1 above)
Comment (Nifty255) [reply lib/Tance.js:3]: Whoops. Result of copy-pasta :cirlaugh: I'll fix it.

**Pull request merged**